src/data/select.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  HassEntityAttributeBase,
+  HassEntityBase,
+} from "home-assistant-js-websocket";
+import { HomeAssistant } from "../types";
+
+interface SelectEntityAttributes extends HassEntityAttributeBase {
+  options: string[];
+}
+
+export interface SelectEntity extends HassEntityBase {
+  attributes: SelectEntityAttributes;
+}
+
+export const setSelectOption = (
+  hass: HomeAssistant,
+  entity: string,
+  option: string
+) =>
+  hass.callService("select", "select_option", {
+    option,
+    entity_id: entity,
```

Comment:
Should this not use `target`?

Reply:
🤷  Doesn't matter much at this point.
None of the internal handlers use target at this point.

Reply:
Now uses a target.